src/monkey_patch/function_modeler.py

```diff
@@ -190,19 +186,16 @@ def _construct_config_from_finetune(self, finetune_hash, finetune):
         nr_of_training_runs = decode_int(next_char) + 1
         nr_of_training_points = (2 ** nr_of_training_runs) * 200
         config = {
-                    "distilled_model": model,
-                    "current_model_stats": {
-                        "trained_on_datapoints": nr_of_training_points,
-                        "running_faults": []},
-                    "last_training_run": {"trained_on_datapoints": nr_of_training_points},
-                    "current_training_run": {},
-                    "teacher_models": ["gpt-4","gpt-4-32k"], # currently supported teacher models
-                    "nr_of_training_runs": nr_of_training_runs}
-            
-        return config
-
-
+            "distilled_model": model,
+            "current_model_stats": {
+                "trained_on_datapoints": nr_of_training_points,
+                "running_faults": []},
+            "last_training_run": {"trained_on_datapoints": nr_of_training_points},
+            "current_training_run": {},
+            "teacher_models": ["gpt-4", "gpt-4-32k"],  # currently supported teacher models
```

Comment:
We should add the biggest available Claude model here. @MartBakler is this correct?